content/en/tracing/faq/service_definition_api.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Service Definition API
+kind: faq
+is_beta: true
+---
+
+<div class="alert alert-warning">This feature is in private beta and the endpoints are likely to change.</div>
+
+Before you begin, you need a [Datadog API and app key][1].
+
+## Post a service definition
+
+```
+POST https://api.datadoghq.com/api/unstable/services/definition
+```
+
+### Arguments
+
+#### Header parameters
+
+| Required field  | Description |
+| ---------- | ----------- |
+| `DD-API-KEY` | Identifies an organization. To create or reuse existing keys, go to the [API keys page][2]. |
+| `DD-APPLICATION-KEY` | Identifies a user. To create or reuse existing keys, to to the [Application keys page][3]. |
+
+### Request
+
+#### Body data (required)
+
+| Field                       | Type            | Description |
+| --------------------------- | --------------- | ------------------------------------------------------- |
+| schema-version&nbsp;[_required_] | string          | Version of the service definition schema being used. Only value `v2` is supported.|
+| dd-service&nbsp;[_required_]     | string          | Unique identifier of the service. Must be unique across all services and it used to match with a service in Datadog. |
+| team                        | string          | Name of the team responsible for the service. |
+| contacts                    | object          | List of contacts on the team. |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;type&nbsp;[_required_]      | string          | Contact type  |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;name                        | string          | Contact name  |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;contact&nbsp;[_required_]   | string          | Contact value |
+| links                       | object          | List of important links related to the service (for example, a runbook). |
```

Comment:
`links` is optional

Reply:
these top level objects are optional: 
* contacts
* links
* docs
* repos
* extensions
* tags
* integrations